Userland/Libraries/LibWeb/PixelUnits.cpp

```diff
@@ -67,7 +67,11 @@ double CSSPixels::to_double() const
 
 int CSSPixels::to_int() const
 {
-    return m_value / fixed_point_denominator;
+    // Truncate on cast
+    return (m_value >> fractional_bits)
+        + ((m_value & radix_mask)
+                ? m_value > 0 ? 0 : 1
+                : 0);
```

Comment:
hmm, could you provide some explanation on why truncation is needed here?

Reply:
or better write test that demonstrates why it is required :)

Reply:
I think the commit title says it, but I can add a comment
but essentially that matches what floats do when you cast them
Will also add a test

Reply:
it doesn't seem like there any reason why would want to match this truncation behaviour that happens for float.